drop-boxes/register-mtb-data-dropbox/register-mtb-data-dropbox.py

```diff
@@ -128,10 +128,10 @@
 SERVICE_CREDENTIALS.user = tracking_helper.get_service_user()
 SERVICE_CREDENTIALS.password = tracking_helper.get_service_password()
 SERVICE_REGISTRY_URL = URL(tracking_helper.get_service_reg_url())
-QBIC_LOCATION = tracking_helper.get_qbic_location_json()
+DATA_AVAILABLE_JSON = tracking_helper.get_data_available_status_json()
 
-### We need this object to update the sample location later
-SAMPLE_TRACKER = SampleTracker.createQBiCSampleTracker(SERVICE_REGISTRY_URL, SERVICE_CREDENTIALS, QBIC_LOCATION)
+### We need this object to update the sample status later
+SAMPLE_TRACKER = SampleTracker.createLocationIndependentSampleTracker(SERVICE_REGISTRY_URL, SERVICE_CREDENTIALS, DATA_AVAILABLE_JSON)
```

Comment:
why do we need to set this in one file two times? You basically overwrite the same variables with the same values

Reply:
duplicate has been removed, thanks